app/assets/javascripts/mobile/mobile_post_actions.js

```diff
@@ -3,6 +3,11 @@
     initialize: function() {
       $(".like-action", ".stream").bind("tap click", this.onLike);
       $(".reshare-action", ".stream").bind("tap click", this.onReshare);
+      // Add handler to newly loaded comments
+      var self = this;
+      $(".stream").bind("comments.loaded", function(evt){
```

Comment:
- 'evt' is defined but never used
- Missing space before opening brace.

Reply:
Sorry for these Style - warnings. I switched to VSCode recently and this 'saves' files on commit. Before this 'pronto' rund well locally. Thats why some style checks still are overseen.

Reply:
Save on commit sounds like an incredibly bad idea, what if you also want to test and run your changes locally before you commit, and then you would still see the old changes? :thinking: 

But I can recommend an automated pre-push hook, so pronto runs before you push and tells you the result (and aborts the push if needed): https://wiki.diasporafoundation.org/Styleguide#Perform_an_automatic_local_review_before_pushing_to_your_repo

Reply:
Ah.. the '--unstaged' option in pronto may be the missing part here.